src/main/java/org/catrobat/jira/timesheet/activeobjects/Timesheet.java

```diff
@@ -65,9 +65,15 @@
         ACTIVE, INACTIVE, AUTO_INACTIVE, INACTIVE_OFFLINE, DISABLED, DONE
     }
 
+    int getTotalHours();
```

Comment:
This will add another database column to this ActiveObject which is not what we want. I would suggest to remove those lines and replace it with the following (at the end of the class):

```
@Ignore
int getTotalHours();
```

Another suggestion would be -- to not mistake it with an actual property of this ActiveObject (i.e., `getHoursCompleted`) like you just did -- to change the name to `calculateTotalHours()` indicating that this is not a getter for a field.

Reply:
added getTotalHours to Ignore and renamed function to calculateTotalHours()